sinonjs/sinon
# Remove traces of legacy IE

#### Purpose
Remove code and docs referring to unsupported versions of Internet Explorer. If this is accepted we can remove/tick off most of the remaining checkboxes of #966 (like `Migrate util/fake-xdomain-request suite`).

#### Background (Problem in detail) 
#1221 pretty much concludes that we support IE11+ for Sinon 2, while #966 still mentions the XDR and XHR code for earlier IE versions. [That seems unnecessary](https://github.com/sinonjs/sinon/issues/966#issuecomment-274250567). To get these points out of the way for a release, let's just delete the code we are not supporting in any case.

#### How to verify - mandatory
1. Check out this branch (see github instructions below)
2. `npm install`
3.  export SAUCE_USERNAME=sinonjs
4. export SAUCE_ACCESS_KEY=<secret - available for sinonjs maintainers>
5. `npm run test-cloud`
